tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/experimental/internal/MatchingStrategies.kt

```diff
@@ -44,17 +45,22 @@ open class UsageCompatibility: AttributeCompatibilityRule<Usage> {
 
         when {
             requested == null -> compatible()
-            requested == Usage.JAVA_API && (provided == Usage.JAVA_API || provided == KotlinNativeUsage.KLIB) -> {
+            requested == provided -> compatible()
```

Comment:
Not sure if this rule is necessary, IIRC it's always implicitly applied.

Reply:
Fixed